tests/yast2_gui/yast2_control_center.pm

```diff
@@ -123,220 +136,292 @@ sub run() {
 
     #   start Online Migration
     assert_and_click "yast2_control-center_online-migration";
-    assert_and_click "yast2_control-center-online-migration_cancel";
+    assert_screen "yast2_control-center-online-migration_cancel", 60;
+    send_key "alt-c";
     assert_screen 'yast2-control-center-ui';
 
     #   start Partitioner
     assert_and_click "yast2_control-center_partitioner";
-    assert_screen "yast2_control-center-partitioner_warning";
+    assert_screen "yast2_control-center-partitioner_warning", 60;
     assert_and_click "yast2_control-center-partitioner_abort";
     assert_screen 'yast2-control-center-ui';
 
     #   start Service Manager
     assert_and_click "yast2_control-center_service-manager";
-    assert_screen "yast2_control-center-service-manager";
+    assert_screen "yast2_control-center-service-manager_list", 60;
     assert_and_click "yast2_control-center-service-manager_ok";
     assert_screen 'yast2-control-center-ui';
 
     #   start Authentication Server
     #	need to scroll down to get other modules
     send_key "down";
     assert_and_click "yast2_control-center_authentication-server";
-    assert_screen "yast2_control-center-authentication-server_install";
-    assert_and_click "yast2_control-center-authentication-server_install-cancel";
-    assert_and_click "yast2_control-center-authentication-server_install-error";
-    assert_screen 'yast2-control-center-ui';
+    if (check_screen "yast2_control-center-authentication-server_install") {
```

Comment:
okay, will create a new needle here. I can even check the results of package installation. It should like following:
--
assert_and_click "yast2_control-center_authentication-server";
assert_screen [qw[yast2_control-center-authentication-server_install yast2_control-center-authentication-server_configuration )];
if (match_has_tag ('yast2_control-center-authentication-server_install')) {
send_key "alt-i";
 }
else {  
send_key "alt-c"; 
}
assert_screen 'yast2-control-center-ui', 60;